server/controllers/court-information-two-controller.ts

```diff
@@ -33,8 +31,9 @@ const courtInformationTwoController = async (
 
     const viewModel = mapCaseDetailsToHearingSummary(caseDetails)
     res.locals.hearingData = viewModel
+    res.locals.caseDetails = caseDetails
 
-    return res.render(view)
+    res.render(view)
```

Comment:
Similarly as above, undo the change and add to the same `card` (as above) for the change.